Description: Fix FTBFS due missing includes
Author: Tobias Frost <tobi@debian.org>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=822499
Forwarded: no
Last-Update: 2016-04-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/file/extractor/autotests/iohandlertest.cpp
+++ b/src/file/extractor/autotests/iohandlertest.cpp
@@ -29,6 +29,8 @@
 
 #include "../iohandler.h"
 
+#include <cmath>
+
 namespace Baloo {
 class IOHandlerTest : public QObject
 {
--- a/src/file/timeestimator.cpp
+++ b/src/file/timeestimator.cpp
@@ -23,6 +23,8 @@
 #include "timeestimator.h"
 #include "filecontentindexerprovider.h"
 
+#include <math.h>
+
 using namespace Baloo;
 
 TimeEstimator::TimeEstimator(QObject* parent)
